t/ui/10-tests_overview.t

```diff
@@ -319,6 +340,17 @@ subtest "filtering by machine" => sub {
     };
 };
 
+subtest "job template names displayed on 'Test result overview' page" => sub {
+    $driver->get('/group_overview/1002');
+    is($driver->find_element('.progress-bar-failed')->get_text(), '1 failed', 'The number of failed jobs is right');
+    is($driver->find_element('.progress-bar-unfinished')->get_text(),
+        '1 unfinished', 'The number of unfinished jobs is right');
+
+    $driver->get('/tests/overview?distri=opensuse&version=13.1&build=0091&groupid=1002');
+    my @tds = $driver->find_elements('#results_DVD tbody tr .name');
+    is($tds[0]->get_text(), 'kde_variant', 'job template name kde_variant displayed correctly');
+};
```

Comment:
I found out that on the job details page the test suite description is correctly shown for both kind but on the test overview page it is not shown for the parameterized job template.

Reply:
Do you mean the description is not shown in `overview` page? our overview dose not show test suite description, right?  I do not see any test suite description is shown in the test over view page. For example `https://openqa.suse.de/tests/overview?distri=sle&version=12-SP5&build=0372&groupid=167`

Reply:
each test row that is corresponding to a test suite shows a link instead of just text. if you click on it you get a popover with the test suited description